src/Umbraco.Core/Composing/IUserComposer.cs

```diff
@@ -1,11 +1,9 @@
-﻿namespace Umbraco.Cms.Core.Composing
+namespace Umbraco.Cms.Core.Composing
 {
     /// <summary>
     /// Represents a user <see cref="IComposer"/>.
     /// </summary>
-    /// <remarks>
-    /// <para>User composers compose after core composers, and before the final composer.</para>
-    /// </remarks>
+    [System.Obsolete("This interface is obsolete. Use IComposer instead.")]
```

Comment:
Let's add a `using System;` above and update to a more helpful message:
```suggestion
    [Obsolete("Composing is always done after core services are configured during startup, use IComposer instead.")]
```

We might want to explain that `ICoreComposer` is removed/replaced with the native DI container... The above message is already quite long, so it should be described more compactly or otherwise be added as remarks/XML documentation.